src/components/tables/tables.scss

```diff
@@ -1,152 +1,197 @@
 @import '../../assets/scss/variables';
 //// Table Elements
 
- table {
+.table-responsive-container {
   width: 100%;
-  font-size: $content-font-size;
-}
+  position: relative;
 
- table {
-  border-spacing: 0;
-  border-collapse: collapse;
-}
+  &.row-headers--true {
+    .track-heading:first-child {
+      left: 0;
+    }
 
- table td,
- table th {
-  padding: $md;
-  vertical-align: top;
-  padding: $md;
-}
+    &.left-sticky-minwidth .track-heading:first-child {
+      position: sticky;
+    }
+  }
 
- table th {
-  color: $secondary;
-}
+  &.is-striped {
+    &.isSticky-Left.left-sticky-minwidth {
```

Comment:
This is a weird mixture of camelCase and kebab-case. Our class naming up to this point has been entirely kebab-case, so I think we should stick with that.

In this case, the line above would be re-written:
```
  &.is-sticky-left.left-sticky-minwidth {
```

And of course, all references to the classes in question would need to be updated.